.github/workflows/gradle.yaml

```diff
@@ -0,0 +1,17 @@
+name: Java CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
```

Comment:
As I mentioned in [a comment on this PR](https://github.com/UMM-CSci-3601/intro-to-git/pull/35#issuecomment-576016675), do we want/need to specify a version number at all? If we do, I would think we'd want to pull this up to v11 since that's what we have in the lab.

Reply:
I ran an experiment where I left it out, and that fails, so GitHub Actions _do_ require a version number. That pretty much resolves this, and I'm going to update the JDK versions to v11.